dotcom-rendering/src/web/layouts/NewsletterSignupLayout.tsx

```diff
@@ -0,0 +1,332 @@
+import { css } from '@emotion/react';
+import type { ArticleFormat } from '@guardian/libs';
+import {
+	brand,
+	brandBackground,
+	brandBorder,
+	brandLine,
+	neutral,
+	space,
+	until,
+} from '@guardian/source-foundations';
+import { SvgGuardianLogo } from '@guardian/source-react-components';
+import { StraightLines } from '@guardian/source-react-components-development-kitchen';
+import { buildAdTargeting } from '../../lib/ad-targeting';
+import { AdSlot, MobileStickyContainer } from '../components/AdSlot';
+import { ArticleBody } from '../components/ArticleBody';
+import { ArticleContainer } from '../components/ArticleContainer';
+import { ContainerLayout } from '../components/ContainerLayout';
+import { ElementContainer } from '../components/ElementContainer';
+import { Footer } from '../components/Footer';
+import { Header } from '../components/Header';
+import { HeaderAdSlot } from '../components/HeaderAdSlot';
+import { Island } from '../components/Island';
+import { Nav } from '../components/Nav/Nav';
+import { NewsletterBadge } from '../components/NewslettersBadge';
+import { OnwardsUpper } from '../components/OnwardsUpper.importable';
+import { SubNav } from '../components/SubNav.importable';
+import { getContributionsServiceUrl } from '../lib/contributions';
+import { decidePalette } from '../lib/decidePalette';
+import { getCurrentPillar } from '../lib/layoutHelpers';
+import { BannerWrapper, Stuck } from './lib/stickiness';
+
+// This Layout is not currently in use.
+// It is an outline of a design for articles with the ArticleDesign.NewsletterSignup
+// which are currently rendered using the standard layout.
+// The full version of the design is to be implemented by the newsletters team.
+
+// to use this layout, edit ./dotcom-rendering/src/web/layouts/DecideLayout.tsx
+// to return is on articles with  ArticleDisplay.Standard && ArticleDesign.NewsletterSignup
+
+const mainColWrapperStyle = css`
+	display: flex;
+	height: 100%;
+	flex-direction: column;
+	justify-content: flex-end;
+	max-width: 200px;
+
+	${until.wide} {
+		max-width: 160px;
+	}
+
+	${until.leftCol} {
+		justify-content: flex-start;
+		max-width: 200px;
+	}
+
+	${until.desktop} {
+		max-width: 170px;
+	}
+`;
+
+const mainColGuardianLogoContainerStyle = css`
+	svg {
+		display: none;
+
+		${until.leftCol} {
+			width: 65%;
+			display: block;
+		}
+	}
+`;
+
+// the negative bottom values at the two column layout are to
+// align the baseline of the text in the newsletters badge svg
+// with the guardaina logo in the leftCol, rather than aligning
+// the bottom of the SVG frame (design requirement)
+const mainColNewsLettersBadgeContainerStyle = css`
+	svg {
+		width: 100%;
+		position: relative;
+		bottom: -10.75px;
```

Comment:
Genuine question: do the decimal pixels make a difference here? Is this something to do with how expensive my screen is?

Reply:
I think it's only noticeable if you zoom in - I assumed people will to be on the safe side. I'm not sure on this, but it might vary between browsers.

Reply:
👍 I've rounded them down to the pixel here.